docs/user-guides/contributors_guide.md

```diff
@@ -51,21 +70,18 @@ After accepting a new feature or confirming that a bug was found, an Issue may b
 People *love* thorough bug reports. 
 

```

Comment:
@EtricKombat  Please use the issue section for hyperledgerkochi/KOBman for the sample issue